kpi/deployment_backends/kc_access/shadow_models.py

```diff
@@ -591,7 +591,7 @@ def absolute_mp3_path(self):
             content = self.get_mp3_content()
             kobocat_storage.save(self.mp3_storage_path, ContentFile(content))
 
-        if not isinstance(kobocat_storage, KobocatS3Boto3Storage):
+        if isinstance(kobocat_storage, KobocatFileSystemStorage):
```

Comment:
I reversed this to make anything not local storage, use the S3 like storage. Which maybe works for azure too. I really hate this but it's the only quick way I could think to make it work. It should really not be so S3 specific and just happen to maybe work in other places.